src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -62,11 +62,15 @@
 (defn create-form-label [text]
   [:div {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+;; jQuery3+React warns about nil :value elements. Filter.
```

Comment:
Required because jquery3+react complained about nil values being assigned to properties on an input element. Does not occur under jquery2+react:
```
`Warning: `value` prop on `input` should not be null. Consider using the empty string to clear the component or `undefined` for uncontrolled components.`
```